src/components/Overview.tsx

```diff
@@ -59,7 +59,25 @@ export default function Overview() {
       },
     },
   ];
-
+  
+  integrations.forEach((integration) => {
+    if (integration.tabs) {
+      integration.tabs.forEach((tab) => {
+        tabs.push(
+          {
+            name: tab.name,
+            active: activeTab === tab.name,
+            onSelect: () => {
+              setEventId(null);
```

Comment:
def gotta figure out a better way to handle routing params.

idk if react-router could work _without_ actually persisting the route in the url or something

or we could just pass around route names/params in some home grown shenanigan

Reply:
Just hacking stuff to make it work now